docs/reference/model.rst

```diff
@@ -108,17 +90,24 @@ Subscription Classes
    SubscriptionCallback
    SubscriptionUpdate
 
-Club / Effort / Performance Classes
-------------------------------------
-   Split
-   Club
-   BaseEffort
-   BestEffort
-   DistributionBucket
-   BaseActivityZone
+.. _custom-types-anchor:
 
-Club Classes
--------------
-   MetaClub
-   SummaryClub
-   DetailedClub
+Custom Types
+--------------------------
+.. autosummary::
+    :toctree: api/
+
+    Distance
+    Duration
+    Timezone
+    Velocity
+
+Stravalib helpers
```

Comment:
I placed these at the bottom. Sphinx will provide warnings if we skip classes. If there is a strong reason to not include these helpers we can consider a manual removal! or just ignoring the warning. but it might be nice to include them for contributors.